opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -69,6 +69,10 @@ class LogDroppedAttributesWarning(UserWarning):
 
 warnings.simplefilter("once", LogDroppedAttributesWarning)
 
+AnyValue = Union[
```

Comment:
We can probably place this in `opentelemetry.util.types`

Reply:
Hi @lzchen ,
Changes are made !
would you review changes please!